Makefile

```diff
@@ -22,7 +22,7 @@ DdevVersion ?= $(VERSION)
 WebImg ?= drud/nginx-php-fpm7-local
 WebTag ?= v0.3.4
 DBImg ?= drud/mysql-docker-local-57
-DBTag ?= v0.2.0
+DBTag ?= v0.2.1
```

Comment:
DBTag [v0.3.0](https://github.com/drud/mysql-docker-local/releases/tag/v0.3.0) was already tagged by one @tannerjfco 3 days ago. Ya gotta keep up! Probably just good to rebase this PR.